ocp_resources/benchmark.py

```diff
@@ -0,0 +1,95 @@
+import logging
+
+from openshift.dynamic.exceptions import NotFoundError
+
+from ocp_resources.resource import NamespacedResource
+from ocp_resources.utils import TimeoutSampler
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Benchmark(NamespacedResource):
+    """
+    Benchmark resource
+    Defined by https://github.com/cloud-bulldozer/benchmark-operator
+
+    The benchmark-operator monitors the `my-ripsaw` namespace for `Benchmark` resources.
+    When a new `Benchmark` is created, the benchmark-operator creates and starts the pods or VMs necessary,
+    and triggers the benchmark run.
+    """
+
+    api_group = NamespacedResource.ApiGroup.RIPSAW_CLOUDBULLDOZER_IO
```

Comment:
self.ApiGroup.RIPSAW_CLOUDBULLDOZER_IO

Reply:
`self` is not exposed here, unless you mean to override `__init__`?

Reply:
Sorry, was confused with something else :)